app/src/main/java/com/asfoundation/wallet/rating/RatingInteractor.kt

```diff
@@ -0,0 +1,63 @@
+package com.asfoundation.wallet.rating
+
+import com.appcoins.wallet.bdsbilling.WalletService
+import com.asfoundation.wallet.transactions.Transaction
+import com.asfoundation.wallet.ui.gamification.GamificationInteractor
+import io.reactivex.Completable
+import io.reactivex.Scheduler
+import io.reactivex.Single
+
+class RatingInteractor(private val ratingRepository: RatingRepository,
+                       private val gamificationInteractor: GamificationInteractor,
+                       private val walletService: WalletService,
+                       private val ioScheduler: Scheduler) {
+
+  fun shouldOpenRatingDialog(): Single<Boolean> {
+    val remindMeLaterDate = ratingRepository.getRemindMeLaterDate()
+    if (remindMeLaterDate > -1F && remindMeLaterDate <= System.currentTimeMillis()) {
```

Comment:
remindMeLaterDate is a long, but you're using a float on the comparision. I think there's no problems, just a matter of keeping the same type

Reply:
👍 I wanted to put "L" instead